Apps/CommonData/src/Validations/DependantAgeValidator.cs

```diff
@@ -31,12 +31,18 @@ public class DependantAgeValidator : AbstractValidator<DateTime>
         /// <param name="maxDependentAge">The maximum age of the dependent.</param>
         public DependantAgeValidator(DateTime? now = null, int maxDependentAge = 12)
```

Comment:
We use the spelling "dependent" rather than "dependant".  I see the dependant spelling showing up in the DependantAgeValidator, DependentAgeValidatorTests, and AddDependentRequestValidator.

Reply:
auto pilot :) https://www.merriam-webster.com/words-at-play/spelling-variants-dependent-vs-dependant
I'll get that fixed.